curriculum/challenges/english/25-front-end-development/quiz-css-animations/66ed8fc9f45ce3ece4053eae.md

```diff
@@ -486,3 +486,464 @@ Which CSS property allows you to set a time before the animation begins?
+
+#### --answer--
+
+`@media (prefers-reduced-motion: reduce)`
+
+### --question--
+
+#### --text--
+
+Inside the `reduced‑motion` media query, which declaration disables transitions?
+
+#### --distractors--
+
+`animation: none;`
+
+---
+
+`transition: remove;`
+
+---
+
+`animation-play-state: paused;`
+
+#### --answer--
+
+`transition: none;`
+
+### --question--
+
+#### --text--
+
+What does the `animation-play-state` property allow you to do?
+
+#### --distractors--
+
+Set how many times the animation repeats
+
+---
+
+Specify how long the animation takes to complete
+
+---
+
+Determine the direction in which the animation plays
+
+#### --answer--
+
+Pause and resume the animation
+
+### --question--
+
+#### --text--
+
+As a general rule, what content should be avoided to reduce seizure risk?
+
+#### --distractors--
+
+Any content that changes color
+
+---
+
+Any movement of elements
+
+---
+
+Any use of the transform property
+
+#### --answer--
+
+Content that flashes more than three times per second
+
+### --question--
+
+#### --text--
+
+Why is the `!important` declaration used in the CSS rules?
+
+#### --distractors--
+
+To prevent other media queries from loading
+
+---
+
+To limit styles to the first child element
+
+---
+
+To debug CSS more easily
+
+#### --answer--
+
+To ensure these rules take precedence over other styles
+
+### --question--
+
+#### --text--
+
+What does `animation-iteration-count: 1 !important;` ensure in CSS?
+
+#### --distractors--
+
+That animations are paused
+
+---
+
+That animations run infinitely
+
+---
+
+That animations reverse direction each cycle
+
+#### --answer--
+
+That any looping animations only play once
+
+### --question--
+
+#### --text--
+
+What type of animation content can trigger seizures in people with photosensitive epilepsy?
+
+#### --distractors--
+
+Slow fading transitions
+
+---
+
+Background-color changes
+
+---
+
+Single-direction transforms
+
+#### --answer--
+
+Rapid flashing or strobing effects that flash more than three times per second
+
+### --question--
+
+#### --text--
+
+Which property is NOT part of the `animation` shorthand?
+
+#### --distractors--
+
+animation-delay
+
+---
+
+animation-timing-function
+
+---
+
+animation-direction
+
+#### --answer--
+
+animation-transform
+
+### --question--
+
+#### --text--
+
+What does the `@keyframes` rule define?
+
+#### --distractors--
+
+The timing function of an animation
+
+---
+
+The default state of an element
+
+---
+
+The media queries for animations
+
+#### --answer--
+
+The sequence of styles at different points in an animation
+
+### --question--
+
+#### --text--
+
+What is the goal of using the `prefers-reduced-motion` media query in animations?
+
+#### --distractors--
+
+To completely remove all motion from the website
+
+---
+
+To force animations to run at double speed
+
+---
+
+To disable all CSS styles globally
+
+#### --answer--
+
+Use a low-motion approach that respects user settings for comfortable accessibility.
```

Comment:
this question is pretty similar to a few other questions from earlier. come up with different one